### DISCUSSION TITLE
portable tomb

### DISCUSSION THREAD

**Discussion by tathastu871** [dyne/tomb#441] [Q&A] [answered]
is it possibel to run tomb portably directly from usb drive irrespective of operative system

**Comment by Narrat** [comment]
As long as it finds all its deps tomb itself can be made somewhat portable. There is also issue https://github.com/dyne/Tomb/issues/380 which wants to support busybox.
But running across the different operating systems is not possible. It may run on Windows with cygwin/msys2 in general, but dunno about cryptsetup which is used for the encryption.
Long story short: **If** it is possible to achieve it still needs heavy tinkering. And heavy doubts about the if, as it relies on tools, which aren't known for being cross platform.

**Comment by jaromil** [comment, marked as answer]
Good news! I have started development of a [portable tomb script](https://github.com/dyne/Tomb/tree/master/portable). A basic test suite is already in place and the README outlines the features, most important to note that it uses Veracrypt and only POSIX shell.
